device/dell/x86_64-dell_z9664f-r0/plugins/eeprom.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+#############################################################################
+# DellEMC Z9664f
+#
+# Platform and model specific eeprom subclass, inherits from the base class,
```

Comment:
Shouldn't we have copyright headers in each of these files?

Reply:
It's not been added to any of the platform files. Hence followed same convention. Please let me know if that is required.